projects/addon-doc/components/aside/aside.component.html

```diff
@@ -0,0 +1,14 @@
+<ng-content></ng-content>
+<nav
+    tuiTabs
+    vertical="right"
+    [(activeItemIndex)]="activeIndex"
+>
+    <button
+        *ngFor="let item of menuList"
+        tuiTab
+        (click)="onView(item.id)"
+    >
+        {{ item.title | tuiDocExampleCapitalize }}
```

Comment:
Is this pipe really necessary? Can we just combine `text-transform: capitalize;` and `::first-letter`? Seems like we should drop this pipe completely and the utility function underneath it in a separate refactor PR.